### PR TITLE
Updated Dockerfile to install Crystal. And submitting my solution

### PR DESCRIPTION
**Pull request by patrickcarver**
I've updated the Dockerfile so that it will install Crystal. I'm not expert on Docker and Crystal is in 0.31, so I cobbled together the command the best I could.

I've included my solution written in Crystal.

Probably should I made this two separate pull requests, but ¯\_(ツ)_/¯

### CODE REVIEW

**Comment by drewpearce** [comment]
You should definitely make this 2 pull requests, since the Dockerfile is no longer in this repo =)

File: Dockerfile [hunk 1]
```diff
@@ -39,6 +39,16 @@ ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 
+# Install Crystal 0.31
```
Comment (drewpearce) [inline Dockerfile]: Docker file is in separate repo -- https://github.com/ReformedDevs/challenge-docker

File: README.md [hunk 2]
```diff
@@ -82,15 +82,16 @@ If you only want to run specific directories on a local run (i.e. just test your
 
 ### Leaderboard
 
-__Inputs__: _aogpnnx, kigvvu, flzcxa, cokbfd, bkgi_
+__Inputs__: _rdaswpf, vnih, nyzhro, wrujd, usjc_
```
Comment (drewpearce) [inline README.md:85]: leaderboard will be updated in standardized droplet after your PR is merged.

File: test_results.json [hunk 3]
```diff
@@ -3,1762 +3,2010 @@
     {
       "Author": "plusuncold",
       "Language": "C++",
-      "Word": "pax, kivu, zax, bock, big",
-      "Score": "12, 11, 19, 12, 6",
-      "Time (ms)": 18.4,
+      "Word": "dwarf, hin, zo, jud, ju",
```
Comment (drewpearce) [inline test_results.json:6]: test_results.json will be updated in standardized droplet after your PR is merged.

**Review by drewpearce — CHANGES_REQUESTED** [review]
Remove .DS_Store file

File: Dockerfile (hunk 1 above)
Comment (patrickcarver) [reply Dockerfile]: Created a pull request at that repo with updated Dockerfile w/ Crystal support.

**Pull request closed without merging**